lang/id/index.md

```diff
@@ -0,0 +1,340 @@
+---
+title: Semantic Versioning 2.0.0
+language: id
+---
+
+Semantic Versioning 2.0.0
+=============================
+
+Ringkasan
+-------
+
+Diberi nomor versi MAJOR.MINOR.PATCH, kenaikan pada:
+
+1. versi MAJOR ketika anda membuat perubahan kompatibilitas API,
+1. versi MINOR ketika anda menambah fungsionalitas pada gaya kompatibilitas sebelumnya dan,
+1. versi PATCH ketika anda membuat perbaikan bug untuk kompatibilitas sebelumnya.
+
+label tambahan untuk pre-release dan build metadata tersedia sebagai ekstensi
+untuk format MAJOR.MINOR.PATCH.
+
+Pengantar
+------------
+
+Dalam dunia manajemen software terdapat tempat menakutkan yang disebut
+"dependency hell" Semakin besar sistem anda tumbuh dan semakin paket anda
+diintegrasikan ke dalam perangkat lunak anda, semakin besar kemungkinan anda
+untuk menemukan diri anda suatu hari pada lingkaran keputusasaan ini.
```

Comment:
Direct translation makes this hard to grasp by Indonesian, try cutting some unnecessary word.